CASMPET-5712 Add steps to ensure that Keycloak user localization has completed. (#1774)
Cray-HPE/docs-csm: operations/security_and_authentication/Keycloak_User_Localization.md

> # Keycloak User Localization

Reviewer: ```suggestion
# Keycloak User Localization Verification
```

> 

Reviewer: ```suggestion
```

> ## Verification Procedure

Reviewer: ```suggestion
```

Verify that the Keycloak users localize job has completed as expected.

> This section can be skipped if user localization is not required.

After an upgrade, it is possible that all expected Keycloak users were not localized. The procedure below helps determine whether or not this has happened, and
provides remediation steps if they are needed.

1. Check to see if the Keycloak users localize job has completed.

   ```bash
   ncn-m002# kubectl -n services wait --for=condition=complete --timeout=10s job/`kubectl -n services get jobs | grep users-localize | awk '{print $1}'`
   ```

   The job completed if the output contains the string `condition met`.

1. If the job completed, check that the count of localized users matches the expected count from the Keycloak server.

   This can be done by looking at the count of users reported from the command below.

   ```bash
   ncn-m002# cray artifacts get wlm etc/passwd /dev/stdout | wc -l
   ```

>    If that count looks correct, then no further action is needed and the remainder of this section should be skipped. Otherwise,

Reviewer: ```suggestion
   If that count looks correct, then no further action is needed and the remainder of this page should be skipped. Otherwise,
```

>    rerun the localize job by following the remaining steps in the section.

Reviewer: ```suggestion
   rerun the localize job by following the remaining steps in this page.
```
